cypress/support/step_definition/global_login_logout/loginLogoutSteps.spec.js

```diff
@@ -0,0 +1,7 @@
+Given('I am logged in as a {string} at {string}', (username, environment) => {
```

Comment:
- currently in main file name is called -> login.js , but it also contains 'logout' custom command in it, 
- so if we keep file name as authentication.js  -> it implies only login because then 'logout' is 'unauthenticating' process, should we keep then 'logout' custom command in a separate file called -> logout.js ?
- or based on above points should I still rename it authentication.js with 'logout' as well in it ?

Reply:
I would say we can adapt another meaningful name in a follow up, because the file name 'loginLogout.js' is the same as in the main already